src/controllers/helpers.js

```diff
@@ -0,0 +1,104 @@
+const {
+  Author, Book, Reader, Genre,
+} = require('../models');
+
+const get404Error = (model) => ({ error: `The ${model} could not be found.` });
+
+const getModel = (model) => {
+  const models = {
+    book: Book,
+    reader: Reader,
+    genre: Genre,
+    author: Author,
+  };
+  return models[model];
+};
+
+const removePassword = (sentData) => {
+  delete sentData.password;
+
+  return sentData;
+};
+
+const chooseModelToInclude = (model) => {
+  if (model === 'book') return { include: [{ model: Genre }, { model: Author }] };
+  if (model === 'genre') return { include: Book };
+  if (model === 'author') return { include: Book };
+
+  return {};
+};
+
+const getAllItems = (res, model) => {
+  const Model = getModel(model);
+
+  Model.findAll({ ...chooseModelToInclude(model) }).then((items) => {
```

Comment:
chooseModelToInclude does what? I think you can name it better to describe what it is doing, like setIncludeStatementByModel?